examples/api-dotenv-request/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog for `api-dotenv-request`
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to the
+[Haskell Package Versioning Policy](https://pvp.haskell.org/).
+
+## Unreleased
+
+## 0.1.0.0 - YYYY-MM-DD
```

Comment:
@juanfcarrillo I don't think we need a Changelog for this example. Let's remove it.